releasenotes/notes/deprecate-visualization-code-8303626254b79abe.yaml

```diff
@@ -0,0 +1,14 @@
+
+deprecations:
+  - |
+    Deprecated visualization code (set pending=False).
+
+    For :func:`.plot_histogram` visualization: Passing :class:`.QuasiDistribution`, 
+    :class:`.ProbDistribution`, or a distribution dictionary to the `data` argument is deprecated.
+    If you would like to plot a histogram from a :class:`.QuasiDistribution`,
+    :class:`.ProbDistribution`, or a distribution dictionary you should be using the
+    :func:`.plot_distribution` function.
+
+    Also removed reverse-bits argument deprecation from 
+    -- :func:`.get_bit_get_index`
+    -- :func:`.get_condition_label_val`
```

Comment:
```suggestion
    Passing a :class:`.QuasiDistribution`,
    :class:`.ProbDistribution`, or a distribution dictionary in for the ``data``
    argument of the :func:`.plot_histogram` visualization function is now
    deprecated. Support for doing this will be removed in the Qiskit 1.0 release.
    If you would like to plot a histogram from a :class:`.QuasiDistribution`,
    :class:`.ProbDistribution`, or a distribution dictionary you should
    use the :func:`.plot_distribution` function instead.
```